### DISCUSSION TITLE
should we remove the nerd emoji?

### DISCUSSION THREAD

**Discussion by raynepaws** [cytrinc/bark-coding#46] [Polls]
![nerd emoji](https://raw.githubusercontent.com/mariocraft987/bark.github.io/0f70aea06f65daef166d448314d5224375205250/src/emojis/svg/nerd.svg)
![kid says that we should change the nerd emoj](https://github.com/mariocraft987/bark.github.io/assets/129226914/dee5b0bc-424c-4c4b-9155-ad9b6ef72936)
i think we should change it or remove it

**Comment by rubiidev18alt** [comment]
it looks too goofy

**Comment by mariocraft987** [comment]
Yes... it would prb be used for bullyng

**Reply by raynepaws** [reply]
that's my concern

**Comment by nmsderp** [comment]
closed cuz it got removed not that long ago
